<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>160</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Müşteri Giriş ve Ürün Seçim Sistemi</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="enterNumberPlate">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>200</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>191</width>
      <height>22</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 11px;  
background-color: rgb(255, 255, 255);
border: 1px solid #afaaa3;  

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Plakanızı giriniz</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="enterPassword">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>240</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;  
border: 1px solid #afaaa3;  </string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Şifrenizi giriniz</string>
    </property>
   </widget>
   <widget class="QPushButton" name="login">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>290</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;  
border: 1px solid #afaaa3;  </string>
    </property>
    <property name="text">
     <string>Giriş Yap</string>
    </property>
   </widget>
   <widget class="QPushButton" name="signUp">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>330</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;  
border: 1px solid #afaaa3;  </string>
    </property>
    <property name="text">
     <string>Kayıt Ol</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eyeButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>240</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>161</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icons/icons/logo2.jpeg);</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>593</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
